frontend_app_workspace_advanced/src/container/WorkspaceAdvanced.jsx

```diff
@@ -241,6 +241,7 @@ export class WorkspaceAdvanced extends React.Component {
   componentDidUpdate (prevProps, prevState) {
     const { state } = this
     // console.log('%c<WorkspaceAdvanced> did update', `color: ${state.config.hexcolor}`, prevState, state)
```

Comment:
I think this commented line can be removed, isn't it? And maybe comment the next line as this log was commented previously?